.github/workflows/docs.yml

```
@@ -0,0 +1,125 @@
+on:
+  pull_request:
+  push:
+    branches: [master]
```

```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review, labeled]
  merge_group:
```
To be consistent with other workflows. Also it's important to have `merge_group:` because some of these checks are `Required`